clinica/utils/inputs.py

```diff
@@ -267,28 +267,52 @@ def find_sub_ses_pattern_path(
 
 
 def _check_information(information: Dict) -> None:
-    if not isinstance(information, dict):
-        raise TypeError("A dict must be provided for the argument 'information'")
-
-    if not all(elem in information for elem in ["pattern", "description"]):
-        raise ValueError(
-            "'information' must contain the keys 'pattern' and 'description'"
+    if not isinstance(information, (dict, list)):
+        raise TypeError(
+            "A dict or list of dicts must be provided for the argument 'information'"
         )
 
-    if not all(
-        elem in ["pattern", "description", "needed_pipeline"]
-        for elem in information.keys()
-    ):
-        raise ValueError(
-            "'information' can only contain the keys 'pattern', 'description' and 'needed_pipeline'"
-        )
+    if isinstance(information, list):
+        for item in information:
+            if not all(elem in item for elem in ["pattern", "description"]):
```

Comment:
Lines 277 to 295 are copy pasted below. Could you turn them into a function (`_check_information_dict`??) and simply call it for all items in the list (`[ _check_information_dict(elt) for elt in information]`)?